qiita_client/plugin.py

```diff
@@ -136,13 +136,46 @@ def __init__(self, name, description, can_be_submitted_to_ebi,
 
 
 class BaseQiitaPlugin(object):
-    def __init__(self, name, version, description, publications=None):
+    # default must be first element
+    _ALLOWED_PLUGIN_COUPLINGS = ['filesystem', 'https']
```

Comment:
What about adding another global variable: `_DEFAULT_PLUGIN_COUPLINGS = 'filesystem'`; then change this to `_ALLOWED_PLUGIN_COUPLINGS = [_DEFAULT_PLUGIN_COUPLINGS, 'https']`.